resource_activity/models/resource_available.py

```diff
@@ -34,40 +39,38 @@ class ResourceAvailable(models.Model):
 
     @api.multi
     def action_reserve(self):
-        for resource_available in self.filtered(lambda record: record.state == "free"):
```

Comment:
Mmmh, I think it might be safer to leave it there. I might have overlooked that in self, there might be resource_available in other states (as the model name does not suggest).